.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "javascript.validate.enable": false
+}
```

Comment:
should be in your personal (global) `.gitignore` if you use vscode

Reply:
That makes sense, I just pushed a commit that should resolve that